Assets/TemporaryTesting/AdditiveLoadTesting.cs

```diff
@@ -0,0 +1,116 @@
+﻿using Microsoft.MixedReality.Toolkit;
```

Comment:
Reminder to delete this file and everything else under TemporaryTesting before checking in.

Reply:
And then put it in a Unity test if possible, can be done in separate PR.

Reply:
Removed.